src/sentry/monitors/models.py

```diff
@@ -583,6 +584,15 @@ def get_last_successful_checkin(self):
             .first()
         )
 
+    def get_incident_grouphash(self):
+        fingerprint = [
+            "monitor",
+            str(self.monitor.guid),
+            self.environment.name,
+            str(self.last_state_change),
+        ]
+        return hash_from_values(fingerprint)
```

Comment:
```suggestion
    @property
    def incident_grouphash(self):
        fingerprint = [
            "monitor",
            str(self.monitor.guid),
            self.environment.name,
            str(self.last_state_change),
        ]
        return hash_from_values(fingerprint)
```

maybe this?

Reply:
Wait why do we even need to do this? We can just generate a unique UUID per incident? the incident row is the source of truth